CHANGELOG.md

```diff
@@ -1,3 +1,49 @@
+# 94.1.0
+
+This minor release intoduces new API to FlexibleHeader, SnackBar and Ripple. It also includes bug fixes for multiple components.
+
```

Comment:
Please call out the new APIs and features in a `## New Features` section here.

Reply:
Done

Reply:
Please see previous "New Features" sections for guidance on what these sections are typically expected to include. Minimally, they should include a short description of the new feature, though a code snippet is also encouraged.